### PR TITLE
No backprop devices

### PR DESCRIPTION
**Pull request by antalszava**
**Context**

As `lighting.qubit` inherits from `default.qubit`, it also inherits the `passthru_devices` entry of the `device.capabilities` dictionary.

This, however, may lead to unexpected behaviour, as the QNode in PennyLane attempts to get a device with the best differentiation method available. Therefore, even without specifying a differentiation method, we may get a `default.qubit.autograd` device.

**Changes**

Removes the `passthru_devices` entry of `device.capabilities`.

### CODE REVIEW

**Comment by josh146** [comment]
Didn't seem to fix the windows wheel :(

File: tests/test_apply.py [hunk 1]
```diff
@@ -454,6 +454,18 @@ def test_load_default_qubit_device(self):
...
+        """Test that lightning.qubit does not support the backprop
+        differentiation method."""
+
+        dev = qml.device("lightning.qubit", wires=2)
+        def circuit():
+            """Simple quantum function."""
+            return qml.expval(qml.PauliZ(0))
+
+        with pytest.raises(qml.QuantumFunctionError):
+            lightning = qml.QNode(circuit, dev, diff_method="backprop")
```
Comment (josh146) [inline tests/test_apply.py:467]:
```suggestion
            qml.QNode(circuit, dev, diff_method="backprop")
```

File: tests/test_apply.py [hunk 2]
```diff
@@ -454,6 +454,18 @@ def test_load_default_qubit_device(self):
...
+
```
Comment (josh146) [inline tests/test_apply.py:468]: Maybe also worth having a test that checks that `diff_method="best"` results in `isinstance(qnode.device, LightningQubit)`? I'm a bit paranoid now 😆

**Review by josh146 — APPROVED** [review]
Thanks for getting onto this so quickly @antalszava! Before merging, don't forget to update the changelog.

File: tests/test_apply.py (hunk 2 above)
Comment (antalszava) [reply tests/test_apply.py:479]: Good idea, added!

**Pull request merged**